src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -420,6 +420,13 @@ private async Task<bool> AreCentralVersionRequirementsSatisfiedAsync()
 
                 return false;
             }
+            IEnumerable<LibraryDependency> packageReferencedDependenciesWithoutCentralVersionDefined = _request.Project.TargetFrameworks.SelectMany(tfm => tfm.Dependencies.Where(d => d.LibraryRange.VersionRange == null));
```

Comment:
line of whitespace between closing brace of above code block and this new variable definition.

Reply:
I did the merge and got a successful CI run. Unfortunately I missed to change the space suggestion above. I will do it next time I will update the file.